draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -226,12 +226,23 @@ not have any effect on connectivity or connection setup time.
 Examples
 ========
 
-The examples below show how the mDNS technique is used during ICE
-processing. The first example shows the typical case, and the 
-other two examples demonstrate how peer-reflexive candidates for
-local IP addresses can be created due to timing differences.
+The examples below show how the mDNS technique is used during ICE processing.
+The first example illustrates an mDNS candidate gathered with a version 4 UUID
+name. The second example shows the typical case of exchanging mDNS candidates,
+and other two examples demonstrate how peer-reflexive candidates for local IP
+addresses can be created due to timing differences.
 
-Normal handling
+mDNS Candidate
+--------------
+
+The exchange of ICE candidates relies on out-of-band signaling, and we
```

Comment:
Might want to combine this into the example below, basically show the candidates exchanged by ICE Agent 1 and ICE Agent 2 in a list below the drawing.

Reply:
Came up with two options for the presentation of the example. Let me know which one reads better.